src/Microsoft.AspNet.Session/SessionDefaults.cs

```diff
@@ -5,9 +5,19 @@
 
 namespace Microsoft.AspNet.Session
 {
+    /// <summary>
+    /// Represents defualts for the Session.
+    /// </summary>
     public static class SessionDefaults
     {
+        /// <summary>
+        /// Represent the default cookie name.
```

Comment:
Should the docs *here* specify the actual default value? I could see there being value in the docs stating what it is so that a reader won't have to look at the source code to figure it out.